src/services/users.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+let users = [];
+let id = 0;
+
+const getAll = () => users;
+
+const create = (name) => {
+  const newUser = {
+    id: id++,
+    name,
+  };
+
+  users.push(newUser);
+
+  return newUser;
+};
+
+const getById = (userId) => {
+  const foundUser = users.find(user => user.id === +userId);
+
+  return foundUser || null;
+};
+
+const remove = (userId) => {
+  users = users.filter(user => user.id !== +userId);
+
+  return users;
+};
```

Comment:
remove should return a boolean dependent on whether something was deleted

Reply:
fixed.